src/sqs-grep.js

```diff
@@ -252,6 +252,17 @@ class SqsGrep {
                     MessageAttributes: options.stripAttributes ? null : message.MessageAttributes,
                 }).promise();
             }
+            // Republish message to it's topic of origin
+            if (options.republish) {
+                const notification = JSON.parse(message.Body);
```

Comment:
This line can throw an exception if the queue has non-JSON messages. My suggestion is that we catch errors and ignore them as those messages are not "re-publishable".